capabilities/communication/send-sms/maps/tyntec.suma

```diff
@@ -36,7 +36,7 @@ map RetrieveMessageStatus {
     response 200 "application/json" {
       // sanity check
       return map error if (!body.history.length) {
-        message = 'no history available'
```

Comment:
Do we check this in tests? - update recordings 🤷

Reply:
It is not tested. As it is not possible to simulate that state on provider.